python/spear/env.py

```diff
@@ -442,6 +460,22 @@ def _get_random_points(self, num_points):
             random_points_serialized, space=Box(low=-np.inf, high=np.inf, shape=(-1,3), dtype=np.float64), byte_order=self._byte_order)
         return random_points
 
+    def _get_reachable_points(self, start_points):
+        assert all(isinstance(i, list) for i in start_points)
+        reachable_points_serialized = self._rpc_client.call("get_reachable_points", start_points)
+        reachable_points = _deserialize_array(
+            reachable_points_serialized, space=Box(low=-np.inf, high=np.inf, shape=(-1,3), dtype=np.float64), byte_order=self._byte_order)
+        return reachable_points
+
+    def _get_trajectories(self, start_points, end_points):
+        assert all(isinstance(i, list) for i in start_points)
+        assert all(isinstance(i, list) for i in end_points)
+        trajectories_serialized_list = self._rpc_client.call("get_trajectories", start_points, end_points)
+        trajectories = np.vectorize(
```

Comment:
Why is this better than a vanilla list comprehension?

Reply:
Updated this